core/clipboard/block_paster.ts

```diff
@@ -56,12 +66,16 @@ export class BlockPaster implements IPaster<BlockCopyData, BlockSvg> {
  * Exported for testing.
  *
  * @param block The block to move to an unambiguous location.
+ * @param coord The initial coordinate to start searching from.
```

Comment:
Maybe it would be better to call this `initialPosition` (or maybe `originalPosition`, since I think it's the position of the copied block that is of interest here) instead of `coord`; "coord" describes the type of the parameter, not it's meaning.